### ISSUE TITLE
New Design not nearly looks as the old.

### ISSUE THREAD

**Issue by mannyhappenings** [mdn/MDN-feedback#35]
It's good to see some change on MDN web docs, as it's the I always refer to whenever I look up anything remotely related to web.

I have been used to old website too much, and probably the reason I cannot look at the new design the same way. For me, MDN web docs has been a source of truth, after RFC docs. In my opinion, the new website looks more like a bad clone of old website with fancy color schemes. Yes, opinions are subjective and I would like to know if other people feel the same. Even if the change was introduced, the old should have been there for sometime instead of a force adoption or may be it was there and I may have missed it.

Suggestion: Please give us a way to access the old MDN docs

**Comment by shljshlj** [comment]
I agree, I find that new color scheme and font size of sidebar make menu items very unreadable.
